packages/frameworks/frameworks.json

```diff
@@ -143,6 +139,34 @@
       }
     }
   },
+  {
+    "name": "Docusaurus",
+    "slug": "docusaurus",
```

Comment:
```suggestion
    "slug": "docusaurus-1",
```

Perhaps we should be specific here?

Reply:
@styfle no, that would break existing configs